apps/desktop/src/lib/stack/types.ts

```diff
@@ -0,0 +1,17 @@
+import { DetailedCommit, VirtualBranch } from '$lib/vbranches/types';
+import { Type } from 'class-transformer';
+
+export class Series {
```

Comment:
It's now called `PatchSeries` on the rust side, let's mirror that name here?

Reply:
Yeah good idea. Do we want to update our vocabulary in general to replace `Series` with `PatchSeries` or only use that in the code? :sweat_smile:

Reply:
Actually - great question, I'm not sure. @krlvi didn't object to me calling it `PatchSeries` in the rust code, and I did that because `Series` on its own is ambiguous, just like `Sequence` or `Group` would be.